dask-kubernetes.yaml

```
@@ -0,0 +1,71 @@
+package:
+  name: dask-kubernetes
+  version: "2025.4.3"
+  epoch: 0
+  description: "Native Kubernetes integration for Dask"
+  copyright:
+    - license: "BSD-3-Clause"
+  dependencies:
+    runtime:
+      - py3.10-packaging
+
+vars:
+  python-version: "3.10"
+  pypi-package: dask_kubernetes
+
+environment:
+  environment:
+    SOURCE_DATE_EPOCH: "315532900"
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - py${{vars.python-version}}-hatch
+      - py${{vars.python-version}}-hatch-bin
+      - py${{vars.python-version}}-hatch-vcs
+      - py${{vars.python-version}}-pip
+      - python-${{vars.python-version}}
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/dask/dask-kubernetes.git
+      tag: ${{package.version}}
+      expected-commit: 547c911efc58fa003d4cb8d49fcd58a7536fa7e7
+
+  - runs: |
+      mkdir -p /usr/share/dask-kubernetes
+      python${{vars.python-version}} -m venv /usr/share/dask-kubernetes/.venv --system-site-packages
+      source /usr/share/dask-kubernetes/.venv/bin/activate
+
+      hatch build
+      pip install dist/dask_kubernetes-${{package.version}}-py3-none-any.whl
+
```

We can now add the path to the virtual environment's site-packages to the system path:

```suggestion

    # Add virtual environment's site-packages to system path
    echo "/usr/share/dask-kubernetes/lib/python${{vars.py-version}}/site-packages" > "/usr/lib/python${{vars.py-version}}/site-packages/${{package.name}}.pth"
```

Now there's no need to export PYTHONPATH anywhere or anything like that